packages/mdc-drawer/_mixins.scss

```diff
@@ -122,3 +122,12 @@
 @mixin mdc-drawer-z-index($value) {
   z-index: $value;
 }
+
+@mixin mdc-drawer-width($width) {
+  width: $width;
+  max-width: $width;
```

Comment:
I know this was already here, but I forget, why'd we need both width and max-width?

Reply:
Good point. Removed.